.packit.yaml

```diff
@@ -20,19 +14,45 @@ upstream_package_name: coreos-installer
 # downstream (Fedora) RPM package name
 downstream_package_name: rust-coreos-installer
 
+packages:
+  coreos-installer-fedora:
+    specfile_path: rust-coreos-installer.spec
+    actions:
+      post-upstream-clone:
+        - wget https://src.fedoraproject.org/rpms/rust-coreos-installer/raw/rawhide/f/rust-coreos-installer.spec
+      changelog-entry:
+        - bash -c 'echo "- New upstream release"'
+  coreos-installer-centos:
+    pkg_tool: centpkg
```

Comment:
Kinda surprised Packit needs to be told this.

Reply:
lol meeee too.